test/runtime/samples/key-block-component-slot/_config.js

```diff
@@ -5,7 +5,7 @@ export default {
 	props: {
 		logs
 	},
-	async test({ assert, component, target, raf }) {
+	async test({ assert, target }) {
```

Comment:
hmm.. these seemed unrelated, maybe can fix them in a separate PR?

Reply:
This is also ok but I need to say that lint is failed without these changes.
I will create new PR about it.